sha1solver/README.md

```diff
@@ -0,0 +1,123 @@
+The program searches for SHA1 collisions. Actually, it replaces bytes at given positions in a given file in order to get desired hash.
+
+The core source code that calculates SHA1 (`sha1.h` and `sha1.c`) comes from [here](https://github.com/clibs/sha1)
+
+### Input file
+
+The file `infile` is an input binary file. The name of the file cannot be changed. 
+
+### Execution and output
+
+You should run multiple instances of the program. Each takes the same input file but different parameters. Each instance writes an answer to the output file `out.txt`, so watch out to not override it. The output file contains text. Each rows contains bytes from the region and the hash of the `infile`.
+
+The expected output (content of the `out.txt` file) contains rows like
```

Comment:
What is the set of parameters that produces this output?

Reply:
I'm trying to achieve the same results.

Reply:
corrected